common/thorhelper/roxiehelper.cpp

```diff
@@ -1555,7 +1555,21 @@ void IRoxieContextLogger::CTXLOGae(IException *E, const char *file, unsigned lin
 
 void HttpHelper::gatherUrlParameters()
 {
-    const char *finger = strchr(urlPath, '?');
+    const char *start = urlPath.str();
+    while (isspace(*start))
```

Comment:
I'm not wild about the idea of setting target as a side effect of calling gatherUrlParameters...